extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -4,12 +4,24 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', [
 	'wikia.log'
 ], function(DOMElementTweaker, log) {
 	'use strict';
-	var logGroup = 'ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory';
+	var logGroup = 'ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory',
+		autoPlayClassName = 'autoplay',
+		videoPlayerClassName = 'video-player',
+		rightVideoPlayerClassName = 'video-player-right';
 
-	function prepareVideoAdContainer(videoAdContainer) {
-		videoAdContainer.style.position = 'relative';
+	function prepareVideoAdContainer(videoAdContainer, params) {
 		DOMElementTweaker.hide(videoAdContainer);
-		videoAdContainer.classList.add('video-player');
+		videoAdContainer.classList.add(videoPlayerClassName);
+
+		if (params.autoPlay) {
+			videoAdContainer.classList.add(autoPlayClassName);
+		}
+
+		if (params.splittedLayoutVideoPosition === 'right') {
```

Comment:
I'm not sure if it should be a part of porvata player. What do you think about moving it to UAP related modules?